draft-opsarea-rfc5706bis.md

```diff
@@ -1241,8 +1241,8 @@ author:
    is not specific to a management protocol, then it should be left
    to data modelers of that protocol to determine how to handle it.
    For example, VLAN identifiers are defined by standard to range
-   from 1 to 4094. Therefore, a YANG vlanid definition representing the
-   12-bit VLAN-ID used in the VLAN Tag header uses a range of "1..4094".
+   from 1 to 4094. Therefore, a YANG "vlan-id" definition representing the
```

Comment:
`vlanid` is the actual object name.  So, I think it should stay, but maybe quote it.

Reply:
But vlan-id is the type.

Reply:
Hmmm, okay.  I initially read this text (prior to this diff) and while I thought vlan-id read better, vlanid made sense in that the context felt like I was defining a leaf vs. a type.  But I'm okay either way.